lib/custom_cops/spec_never_runs.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Getsafe
```

Comment:
I put it in a `Getsafe` module, but I don't have a real opinion either way. CustomCops?🤷

Reply:
Whatever the module is, let's keep it consistent with other custom cops. I believe Alecs created some not long ago, and I think it would be great to have the same module both for custom cops in nxt_cops and in services themselves (like the one you created today).

I like `NxtCops` more, since we're using the `nxt` prefix everywhere anyway.